### PR TITLE
Fix map radius search 

### PR DESCRIPTION
**Pull request by rygim**
- [x] @joeferner
- [ ] @kunklejr
- [x] @mwizeman @dsingley @EvanOxfeld 
- [x] @joeybrk372 @sfeng88 @rygim @jharwig 

CHANGELOG
Fixed: Corrected the calculation for detecting the clicked center of the search radius on map

The way that the center radius was being calculated for both chrome + ff
used object properties that either were there before and were removed
with new open street maps or were possibly how google calculated it.
Now, the shape of the object is detected and the appropriate calculation
for detecting the center of the map search is used.

Fixes issue https://github.com/v5analytics/visallo-lts/issues/1724

### CODE REVIEW

File: web/war/src/main/webapp/js/map/map.js [hunk 1]
```diff
@@ -667,8 +667,19 @@ define([
...
-                    var offset = self.$node.offset(),
-                        centerCoordinate = map.getCoordinateFromPixel([evt.pageX - offset.left, evt.pageY - offset.top]),
+                    var pixelX, pixelY;
+
+                    if (evt.pageX) {
+                        var offset = self.$node.offset();
+                        pixelX = evt.pageX - offset.left;
+                        pixelY = evt.pageY - offset.top;
+                    }
+                    else if (evt.pixel){
```
Comment (jharwig) [inline web/war/src/main/webapp/js/map/map.js]: This handler is a regular jquery event, so pageX/pageY should be used. What is `pixel`?

Comment (jharwig) [inline web/war/src/main/webapp/js/map/map.js]: My mistake it is from open layers. But it should probably be

`var pixel = map.getEventPixel(evt)`

I would check the pixel result and make sure you don't also need to adjust by `offset`

Comment (jharwig) [inline web/war/src/main/webapp/js/map/map.js]: it looks like it is relative to the map, but just verify that the x isn't +~30 (menubar width)

Comment (rygim) [inline web/war/src/main/webapp/js/map/map.js]: changed as per our discussion and verified

**Comment by joeferner** [comment]
I ran across this issue today and fixed it slightly different, before I realized you had a PR.

`evt` has `evt.coordinate` which replaces the need for `pageX`, `pageY`, etc. Much simpler.

**Comment by jharwig** [comment]
@rygim Please update with just evt.coordinate like joe suggested. Don't need any offsets, or pageX references.

**Comment by rygim** [comment]
changed

**Comment by joeferner** [comment]
LGTM... squash the commits before merge

**Pull request merged**